### DISCUSSION TITLE
ScriptReference/RuntimeInitializeOnLoadMethodAttribute-loadType

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#415] [Notes]
# ScriptReference/RuntimeInitializeOnLoadMethodAttribute-loadType

https://docs.unity3d.com/ScriptReference/RuntimeInitializeOnLoadMethodAttribute-loadType.html

**Comment by hybridherbst** [comment]
For others wondering about the event order, they are called in this order:  
static constructor  
SubsystemRegistration  
AfterAssembliesLoaded  
BeforeSplash  
BeforeSceneLoad  
regular Awake()  
regular OnEnable()  
AfterSceneLoad  
Default (without type attribute)  
regular Start()
